diff --git a/features/bootstrap/FeatureContext.php b/features/bootstrap/FeatureContext.php
index dbef307..2afcace 100644
--- a/features/bootstrap/FeatureContext.php
+++ b/features/bootstrap/FeatureContext.php
@@ -23,6 +23,8 @@ require_once __DIR__.'/../../vendor/phpunit/phpunit/PHPUnit/Framework/Assert/Fun
  */
 class FeatureContext extends MinkContext
 {
+    static private $app;
+
     /**
      * Initializes context.
      * Every scenario gets its own context object.
@@ -76,4 +78,44 @@ class FeatureContext extends MinkContext
     {
         return $this->getSession()->getPage();
     }
+
+    /**
+     * @static
+     * @BeforeSuite
+     */
+    static public function bootstrapSilex()
+    {
+        if (!self::$app) {
+            self::$app = require __DIR__.'/../../app/bootstrap.php';
+        }
+
+        return self::$app;
+    }
+
+    private function createAdminUser($username, $password)
+    {
+        return self::$app['user_repository']->createAdminUser(
+            $username,
+            $password
+        );
+    }
+
+    private function createProduct($name, $price)
+    {
+        return self::$app['product_repository']->createProduct(
+            $name,
+            $price
+        );
+    }
+
+    private function updateProduct(\RaptorStore\Product $product)
+    {
+        self::$app['product_repository']->update($product);
+    }
+
+    public function clearData()
+    {
+        self::$app['user_repository']->emptyTable();
+        self::$app['product_repository']->emptyTable();
+    }
 }
