diff --git a/play.php b/play.php
index 0dabca6..556f094 100644
--- a/play.php
+++ b/play.php
@@ -19,7 +19,11 @@ $page = $session->getPage();
 /** @var \Behat\Mink\Element\NodeElement $h1 */
 $h1 = $page->find('css', 'h1');
 
-$registerAnchor = $page->find('css', '.btn-register');
+// $registerAnchor = $page->find('css', '.btn-register');
+$registerAnchor = $page->find(
+    'named',
+    array('link', $session->getSelectorsHandler()->xpathLiteral('REGISTER'))
+);
 $registerAnchor->click();
 
 var_dump($session->getCurrentUrl());
